c_sharp_json_diff/DoubleDiffProcessor.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace c_sharp_json_diff
+{
+    class DoubleDiffProcessor : DiffProcessor
+    {
+        public override Dictionary<string, object> PerformDiffProcess(JToken left, JToken right)
+        {
+            string leftValueAsString = left.ToString();
+            string rightValueAsString = right.ToString();
+            Dictionary<string, object> dict = new Dictionary<string, object>();
+
+            if (!IsIntable(leftValueAsString, rightValueAsString))
```

Comment:
Add a property to the class, that controls how this class processes ints. For example, it could be called ``IntProcessing``. We could make it an enum with these possible values:

1. IgnoreInts - If both left and right are ints, the processor skips that node
2. ProcessIfFraction - It will process left and right if they are both ints but if left does not perfectly divide right
3. ProcessAll - It will process left and right if they are ints, regardless